pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -123,9 +121,22 @@ class Stencil {
   // Number of threads
   unsigned numThreads;
 
+  // The throughput and startup cost of M*N*K matrix multiplication
+  std::pair<double, unsigned> Throughput(const unsigned *ranges,
+                                         const unsigned count);
+
+  // Whether the copster function was set
+  bool costerSet;
+  // External coster function.
+  std::function<std::pair<double, unsigned>(const unsigned *, const unsigned)>
+      coster;
+
 public:
-  explicit Stencil(mlir::AffineParallelOp op, int numThreads)
-      : op(op), numThreads(numThreads) {
+  explicit Stencil(mlir::AffineParallelOp op, int numThreads, bool costerSet,
```

Comment:
Definitely should not have 'costerSet' here, and in fact I'd imagine you should assert coster is a valid callable std::function, since constructing a stencil without a cost function makes no sense.